common/src/main/scala/explore/schemas/implicits.scala

```diff
@@ -0,0 +1,26 @@
+package explore.schemas
+
+import clue.data.syntax._
+import lucuma.core.model.Magnitude
+import ObservationDB.Types.MagnitudeInput
+import java.math.MathContext
+import UserPreferencesDB.Types.WidthUpsertInput
+import UserPreferencesDB.Types.ExploreResizableWidthInsertInput
+
+object implicits {
```

Comment:
Implicits have to be defined outside of the template objects, since they refer to generated types that don't exist before the first generation. This prevents the code from compiling, and scalafix won't run until the code compiles.